go-controller/pkg/cluster/management-port_linux.go

```diff
@@ -78,16 +78,19 @@ func CreateManagementPort(nodeName, localSubnet string, clusterSubnet []string)
 	// source protocol address to be in the Logical Switch's subnet.
 	_, _, err = util.RunIP("neigh", "add", routerIP, "dev", interfaceName, "lladdr", routerMAC)
 	if err != nil && os.IsNotExist(err) {
-		return err
+		return nil, err
 	}
 
 	// Set up necessary iptables rules
 	err = addMgtPortIptRules(interfaceName, interfaceIP)
 	if err != nil {
-		return err
+		return nil, err
 	}
 
-	return nil
+	annotations := make(map[string]string)
+	annotations[OvnNodeManagementPortMacAddress] = macAddress
```

Comment:
@mccv1r0 you can just do:

`return map[string]string{OvnNodeManagementPortMacAddress: macAddress}, nil`

here.